pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -545,45 +553,47 @@ def _save_last_checkpoint(self, trainer, pl_module, ckpt_name_metrics, filepath)
                 trainer.current_epoch,
                 trainer.global_step,
                 ckpt_name_metrics,
-                prefix=self.prefix
+                prefix=self.prefix,
             )
-            last_filepath = os.path.join(self.dirpath, f"{last_filepath}{self.FILE_EXTENSION}")
+            last_filepath = os.path.join(self.dirpath, f"{last_filepath}{self.FILE_EXTENSION}")  # type: ignore
 
         accelerator_backend = trainer.accelerator_backend
 
         if accelerator_backend is not None and accelerator_backend.rpc_enabled:
             # RPCPlugin manages saving all model states
-            accelerator_backend.ddp_plugin.rpc_save_model(self._save_model, last_filepath, trainer, pl_module)
+            accelerator_backend.ddp_plugin.rpc_save_model(  # type: ignore
+                self._save_model, last_filepath, trainer, pl_module
+            )
         else:
             self._save_model(last_filepath, trainer, pl_module)
         if (
-                self.last_model_path
-                and self.last_model_path != last_filepath
-                and (self.save_top_k != -1 or self.save_last)
-                and trainer.is_global_zero
+            self.last_model_path
+            and self.last_model_path != last_filepath
+            and (self.save_top_k != -1 or self.save_last)
+            and trainer.is_global_zero
```

Comment:
> Hello @hemildesai! Thanks for updating this PR.
> 
> * In the file [`pytorch_lightning/callbacks/model_checkpoint.py`](https://github.com/PyTorchLightning/pytorch-lightning/blob/8ee02f8eee840fb2107bc41f7da1c7bbd10357b8/pytorch_lightning/callbacks/model_checkpoint.py):
> 
> > [Line 571:13](https://github.com/PyTorchLightning/pytorch-lightning/blob/8ee02f8eee840fb2107bc41f7da1c7bbd10357b8/pytorch_lightning/callbacks/model_checkpoint.py#L571): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> > [Line 572:13](https://github.com/PyTorchLightning/pytorch-lightning/blob/8ee02f8eee840fb2107bc41f7da1c7bbd10357b8/pytorch_lightning/callbacks/model_checkpoint.py#L572): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> > [Line 573:13](https://github.com/PyTorchLightning/pytorch-lightning/blob/8ee02f8eee840fb2107bc41f7da1c7bbd10357b8/pytorch_lightning/callbacks/model_checkpoint.py#L573): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> 
> ##### Comment last updated at 2020-12-13 08:10:31 UTC

@tchaton `W503` is in the ignore list in `.pep8speaks.yml` so not sure why it's showing up here. I am using `Black` as the formatter and that seems to format it in this way.